conf/messages.en

```diff
@@ -160,3 +160,9 @@ traderDetails.holderOfTransit.name.checkYourAnswersLabel = Transit holder’s na
 traderDetails.holderOfTransit.name.error.required = Enter the transit holder’s name
 traderDetails.holderOfTransit.name.error.length = The transit holder’s name must be 70 characters or less
 traderDetails.holderOfTransit.name.error.invalid = The transit holder’s name must only include letters a to z without accents, numbers 0 to 9, ampersands (&), apostrophes, asterisks(*), forward slashes, full stops, hyphens, percent signs, question marks and spaces
+
+traderDetails.holderOfTransit.addContact.title = Do you want to add a contact?
+traderDetails.holderOfTransit.addContact.heading = Do you want to add a contact?
+traderDetails.holderOfTransit.addContact.checkYourAnswersLabel = Do you want to add a contact?
+traderDetails.holderOfTransit.addContact.error.required = Select yes if you want to add a contact
+traderDetails.holderOfTransit.addContact.error.hidden = if you want to add a contact
```

Comment:
I believe this should be `.change.hidden` instead of `.error.hidden`. This will eventually be used for the check your answers page